package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "clean": "rimraf build",
     "start": "yarn test",
-    "start:watch": "yarn test --watch",
+    "start:watch": "yarn test --watch --reporters=\"default\"",
```

Comment:
Why is this?

Reply:
I had to change it, cause the tests were going into infinite loop!!